contracts/mocks/token/ERC1363ReceiverMock.sol

```diff
@@ -24,9 +24,9 @@ contract ERC1363ReceiverMock is IERC1363Receiver {
         _error = RevertType.None;
     }
 
-    function setUp(bytes4 retval, RevertType err) public {
+    function setUp(bytes4 retval, RevertType error) public {
         _retval = retval;
-        _error = err;
+        _error = error;
```

Comment:
@Amxx I changed this because `error` is a reserved word. Not a big deal but I think I prefer `err`

Reply:
If we want to rename this, then we shoudl also rename the other places (ERC1363SpenderMock for example) for consistency.

Reply:
Ah, I didn't know it was on other mocks.

I changed it initially fearing that something (e.g. transpiler) may break due to the reserved keyword.

Solving